janusgraph-lucene/src/test/java/org/janusgraph/diskstorage/lucene/BerkeleyLuceneTest.java

```diff
@@ -57,4 +62,36 @@ public boolean supportsWildcardQuery() {
     protected boolean supportsCollections() {
         return false;
     }
+
+    @Test
+    public void testPrintSchemaElements() {
+        GraphOfTheGodsFactory.load(graph);
+        mgmt = graph.openManagement();
+
+        String expected = "99EB9D82CE85D15A9120E31969B3F861";
+        String output = mgmt.printSchema();
+        String outputHash = DigestUtils.md5Hex(output).toUpperCase();
+        assertEquals(outputHash, expected);
```

Comment:
Would it be too much work to do more fine-grained asserts? Right now, we'll only get the result that two hashes aren't equal if anything doesn't work as expected which could mean that just the formatting changed a bit or even that the output is completely empty. Maybe you could for example check these aspects with relatively low effort:
* The number of lines should be the number of schema elements + head line + dash lines
* The names of all schema elements should be included in the string.
Any way, that's just a suggestion. You can also keep it as is if you prefer it that way.

Reply:
Number of lines would be really easy to add and I can also pass an array of expected strings I expect output to contain. I still want to keep the hash compare though. We never change graph of gods and if did all it would take is copy the new hash from test failure into the test to update the "mock". I'll add both of these checks when I update to a parameterized test. Thanks for the input.

Reply:
@FlorianHockmann I actually can't apply my proposed fix this until we switch to junit 5 and that won't be coming to the 0.2 branch. I just opened up issue #1393 for myself to update the test after both PRs merged. I think I'm happy enough with this for 0.2 and would like to merge it if you agree.